### DISCUSSION TITLE
GPIO Expansion for SqueezeAMP

### DISCUSSION THREAD

**Discussion by alidaf** [sle118/squeezelite-esp32#371] [Q&A] [answered]
I've started designing a case for the SqueezeAMP but need more GPIOs to accommodate a [rotary dial](https://www.adafruit.com/product/5001) that includes 5 buttons as well as a display, IR and LED. The possibility is alluded to on the SqueezeAMP site but I don't know of any particular product that would work. Would an MCP23017 based board work via I2C? How would this work in the config?

**Comment by Renber78** [comment]
Hello, I suggest you read the documentation because all the answers to your questions are there.

**Reply by alidaf** [reply]
Which documentation in particular - information is spread across different sites, which makes finding specific information harder. I remember seeing something regarding expanders but now can't find it. I've been all over the SqueezeAMP github site.

**Comment by Renber78** [comment, marked as answer]
https://github.com/sle118/squeezelite-esp32/blob/master-v4.3/README.md#gpio-expanders

**Reply by alidaf** [reply]
Thank you.
